### ISSUE TITLE
show the linked items in top of search results

### ISSUE THREAD

**Issue by emilvincentz** [acterglobal/a3#1123]
When going to link a chat/space to a space,  the already linked chats/spaces should figure in the top of the search result list.

**Comment by gnunicorn** [comment]
search of the linking dialog? Or where?

**Comment by emilvincentz** [comment]
yes, so that you can see that chats/spaces that are already linked at the top of the search list.

**Comment by kumarpalsinh25** [comment]
@emilvincentz 
Is this still exists?

**Comment by emilvincentz** [comment]
yes still exists, linked spaces doesnt show on top of the list
